CMakeLists.txt

```diff
@@ -132,7 +134,7 @@ option(NEW_PARSER_DEBUG "Enable tracing of new parser" OFF)
 option(BUILD_MULTI_CORE "Enable building for multicore system" ON)
 option(FAIL_MISSING "Fail when a required external dependency is not present (useful for packagers)" OFF)
 option(USE_GETTEXT "Use the Gettext internationalization library" ON)
-option(BUILD_STATIC_LIBRARY "Also build a static version of the csound library" OFF)
+option(BUILD_STATIC_LIBRARY "Also build a static version of the csound library" ON)
```

Comment:
Any particular reason to default this to ON?

Reply:
Oops, that was just for testing, turned if back off